<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ORDER">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>결제</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../오더.PNG</normaloff>../../오더.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>900</width>
     <height>800</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btPepero">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>211</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>snack_picture/pepero/pepero.svg</normaloff>snack_picture/pepero/pepero.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>280</width>
     <height>500</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btChip">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>snack_picture/potato/potato_chip .png</normaloff>snack_picture/potato/potato_chip .png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>500</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CANCEL">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../취소.PNG</normaloff>../../취소.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>900</width>
     <height>800</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼엑스포</family>
     <pointsize>48</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; color:#00aa7f;&quot;&gt;과&lt;/span&gt;&lt;span style=&quot; font-size:36pt; color:#55aa7f;&quot;&gt;자 &lt;/span&gt;&lt;span style=&quot; font-size:36pt; color:#5500ff;&quot;&gt;자&lt;/span&gt;&lt;span style=&quot; font-size:36pt; color:#00007f;&quot;&gt;판&lt;/span&gt;&lt;span style=&quot; font-size:36pt; color:#ff00ff;&quot;&gt;기&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="CHOCOPRICE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼엑스포</family>
     <pointsize>36</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#322f2f;&quot;&gt;1,500원&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="POKAPRICE">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼엑스포</family>
     <pointsize>36</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#393939;&quot;&gt;1,500원&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbChipCnt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼편지체</family>
     <pointsize>36</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;감자칩 0개&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPeperoCnt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼편지체</family>
     <pointsize>36</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;빼빼로 0개&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#005500;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="TOTALPRICE">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼편지체</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;결제 금액 &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;0원&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAdmin">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>snack_picture/Pngtreevector document setting icon_4225281.png</normaloff>snack_picture/Pngtreevector document setting icon_4225281.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Documents/카카오톡 받은 파일/SNACK_~1/snack_picture/Pngtreevector document setting icon_4225281.png</normaloff>../../../Documents/카카오톡 받은 파일/SNACK_~1/snack_picture/Pngtreevector document setting icon_4225281.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>CANCEL</zorder>
  <zorder>ORDER</zorder>
  <zorder>btPepero</zorder>
  <zorder>btChip</zorder>
  <zorder>label</zorder>
  <zorder>CHOCOPRICE</zorder>
  <zorder>POKAPRICE</zorder>
  <zorder>lbChipCnt</zorder>
  <zorder>lbPeperoCnt</zorder>
  <zorder>btAdmin</zorder>
  <zorder>TOTALPRICE</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
